notebooks/deconvo_sanity_checks.py

```diff
@@ -63,6 +65,9 @@
     random_state=42,
 )
 adata = adata[cell_types_test, :]
+# WARNING. The following line appears in the R sanity checks script:
+# scRNseq_test <- NormalizeData(object = scRNseq_test, normalization.method = "RC",scale.factor = 10000)
+# Should we thus add it here ? (on the raw counts I guess)
 
```

Comment:
Not sure I understand this comment. Thus normalization is applied before the computation of the signature matrix int he R script ?

Reply:
do we know what "RC" means ? in our case, the normalization is done in scanpy= `sc.pp.normalize_total(adata, target_sum=1e4)`

Reply:
Ok. Is this normalization is done in the R scripts that builds the signature matrix? if yes, why should we add it here ?

Reply:
(To normalize the pseudobulk dataset with the same normalization?)

Reply:
Normally, the same R scripts are used to create the signature matrix and then perform the sanity checks. Therefore, the normalization is applied on the test data used for the sanity checks.